packages.json

```diff
@@ -15649,16 +15649,28 @@
     "license": "MIT",
     "web": "https://github.com/lamartire/anycase"
   },
+	{
```

Comment:
Why this got indented?

Reply:
Sorry. Committed with fix indention. thanks